modules/030-cloud-provider-azure/hooks/azure_cluster_configuration_test.go

```diff
@@ -159,44 +134,22 @@ data:
-			Expect(f.ValuesGet("cloudProviderAzure.internal.providerClusterConfiguration.sshPublicKey").String()).To(Equal("ssh-rsa AAA"))
-			Expect(f.ValuesGet("cloudProviderAzure.internal.providerClusterConfiguration.vNetCIDR").String()).To(Equal("10.50.0.0/16"))
-			Expect(f.ValuesGet("cloudProviderAzure.internal.providerClusterConfiguration.subnetCIDR").String()).To(Equal("10.50.0.0/24"))
-			Expect(f.ValuesGet("cloudProviderAzure.internal.providerClusterConfiguration.standard.natGatewayPublicIpCount").String()).To(Equal("1"))
-
-			Expect(f.ValuesGet("cloudProviderAzure.internal.providerClusterConfiguration.masterNodeGroup.replicas").String()).To(Equal("1"))
-			Expect(f.ValuesGet("cloudProviderAzure.internal.providerClusterConfiguration.masterNodeGroup.zones").String()).To(MatchJSON(`["1","2","3"]`))
-
-			Expect(f.ValuesGet("cloudProviderAzure.internal.providerClusterConfiguration.masterNodeGroup.instanceClass.machineSize").String()).To(Equal("Standard_F2"))
-			Expect(f.ValuesGet("cloudProviderAzure.internal.providerClusterConfiguration.masterNodeGroup.instanceClass.urn").String()).To(Equal("Canonical:UbuntuServer:18.04-LTS:18.04.202010140"))
-			Expect(f.ValuesGet("cloudProviderAzure.internal.providerClusterConfiguration.masterNodeGroup.instanceClass.diskSizeGb").String()).To(Equal("50"))
-			Expect(f.ValuesGet("cloudProviderAzure.internal.providerClusterConfiguration.masterNodeGroup.instanceClass.diskType").String()).To(Equal("StandardSSD_LRS"))
+			Expect(f.ValuesGet("cloudProviderAzure.internal.providerDiscoveryData.vnetName").String()).To(Equal("test"))
+			Expect(f.ValuesGet("cloudProviderAzure.internal.providerDiscoveryData.subnetName").String()).To(Equal("test"))
+			Expect(f.ValuesGet("cloudProviderAzure.internal.providerDiscoveryData.resourceGroupName").String()).To(Equal("test"))
+			Expect(f.ValuesGet("cloudProviderAzure.internal.providerDiscoveryData.instances.urn").String()).To(Equal("test:test:test"))
+			Expect(f.ValuesGet("cloudProviderAzure.internal.providerDiscoveryData.instances.diskType").String()).To(Equal("test"))
+			Expect(f.ValuesGet("cloudProviderAzure.internal.providerDiscoveryData.instances.additionalTags").String()).To(Equal("{}"))
 
-			Expect(f.ValuesGet("cloudProviderAzure.internal.providerClusterConfiguration.masterNodeGroup.instanceClass.additionalTags.node").String()).To(Equal("master"))
-
-			Expect(f.ValuesGet("cloudProviderAzure.internal.providerClusterConfiguration.nodeGroups.0.name").String()).To(Equal("static"))
-			Expect(f.ValuesGet("cloudProviderAzure.internal.providerClusterConfiguration.nodeGroups.0.replicas").String()).To(Equal("1"))
-			Expect(f.ValuesGet("cloudProviderAzure.internal.providerClusterConfiguration.nodeGroups.0.zones").String()).To(MatchJSON(`["1","2","3"]`))
-
-			Expect(f.ValuesGet("cloudProviderAzure.internal.providerClusterConfiguration.nodeGroups.0.instanceClass.machineSize").String()).To(Equal("Standard_F2"))
```

Comment:
why we remove this tests?

Reply:
These values are only used in terraform, so there is no point in testing them here.

Reply:
ok